src/Settings.php

```diff
@@ -243,6 +243,13 @@ public function __construct() {
 						'id'   => 'enforce_cookies_policy',
 						'type' => 'checkbox',
 					],
+
```

Comment:
The menu item is still "Cookies Text", even though that applies to only 1 out of 3 options. We could rename it to just "Cookies" to be more accurate.

Reply:
As per the discussion on JIRA comment updated as "Cookies Bar Text".